jni/patches/cadical-shared.patch

```diff
@@ -1,31 +1,34 @@
 diff --git a/configure b/configure
-index e761d26..43320d2 100755
+index c02b197..ba0c99b 100755
```

Comment:
Why the commit numbers changed? Have you reconstructed the patch manually from the sources?

Reply:
Yeah, I made a patch from the last version of cadical sources